docs/user-guide/docs-source/appendix-contributing.md

```diff
@@ -0,0 +1,98 @@
+# Appendix: Contributing to Ambit Development
+
+This section provides important information for anyone who would like to contribute to the development of the Ambit tools or to make custom modifications to Ambit for their own purposes. It covers technical setup and developer workflow. For general contributing guidelines and process, see the repository's [CONTRIBUTING.md file](https://github.com/aws-samples/aws-ambit-scenario-designer-ue4/blob/main/CONTRIBUTING.md).
+
+> ✏️ This guide assumes you already have Unreal Engine development experience.
+
+
+
+## Setup
+
+Ensure you have completed the main [Setup > Initial Setup (One-Time)](../setup/#initial-setup-one-time) instructions from this user guide.
+
+In addition to having Unreal Engine 4.27 installed, you will also need the Microsoft Visual Studio development tools required for UE4 C++ development. If you need help with these setup steps, refer to the Unreal Engine 4 documentation, especially ["Setting Up Visual Studio for Unreal Engine"](https://docs.unrealengine.com/4.26/en-US/ProductionPipelines/DevelopmentSetup/VisualStudioSetup/).
+
+Create a new Unreal Engine C++ project to use as a host project while you work on Ambit development. There are no special configuration requirements for this project. The "Blank" project template with default settings will work just fine, but you can also customize the settings if you'd like.
+
+Clone this repository into the "Plugins" folder of the UE project you created. Note, you will need to create the "Plugins" folder first.
```

Comment:
Should swap these two sentences. Instruct to create, if not present, the "Plugins" folder, and then clone into the repo. 
______________________
"this repository" Should link back to the main Ambit repository. Users should able to get to the User Documents without being conscious of the repository. Alternatively, can place a link elsewhere for the repository link.